spec/features/auth_spec.rb

```diff
@@ -16,4 +16,15 @@
     expect(page).to have_content("user@example.com")
   end
 
+  scenario 'When user registers, should be logged in automatically' do
+    visit root_path
+    click_on 'Register'
+    fill_in 'Name', with: 'Joe Smith'
+    fill_in 'Email', with: 'joesmith@email.com'
+    fill_in 'Password', with: 'jsmithpassword'
+    fill_in 'Confirm', with: 'jsmithpassword'
+    within('.registration-form') { click_on 'Register'}
```

Comment:
Could consider a blank line after this one to improve readability by separating setup from assertions.